backend/scripts/printful/getSizeGuide.js

```diff
@@ -1,5 +1,8 @@
-const fetch = require('node-fetch')
 const fs = require('fs')
```

Comment:
nit: add a comment explaining what this script is for and how to use it. same thing for script printful2.js below.